chart/templates/deployment.yaml

```diff
@@ -24,6 +24,9 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "fusionauth.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+        {{- with .Values.podLabels }}
```

Comment:
Please add the default and a comment to `values.yaml`, similar to `podAnnotations`.